packages/ndk-multilib/build.sh

```diff
@@ -44,7 +45,7 @@ prepare_libs() {
 add_cross_compiler_rt() {
 	RT_PREFIX=$NDK/toolchains/llvm/prebuilt/linux-x86_64/lib64/clang/*/lib/linux
 	RT_OPT_DIR=$TERMUX_PREFIX/opt/ndk-multilib/cross-compiler-rt
-	LIBLLVM_VERSION=$(. $TERMUX_SCRIPTDIR/packages/libllvm/build.sh; echo $TERMUX_PKG_VERSION)
+	LIBLLVM_VERSION=$(. $TERMUX_SCRIPTDIR/packages/libllvm/build.sh; echo $LLVM_MAJOR_VERSION)
```

Comment:
Switched this package over to use the new LLVM major version symlink too, since clang and libcompiler-rt will almost certainly be installed already. I will switch Swift over in a separate commit once this package is built, as the two will time out the CI if built together.